auth_saml/security/ir.model.access.csv

```diff
@@ -0,0 +1,6 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_auth_saml_provider,auth_saml_provider,model_auth_saml_provider,base.group_system,1,1,1,1
+access_auth_res_users_saml,auth_res_users_saml,model_res_users_saml,base.group_system,1,1,1,1
```

Comment:
Wanted to point this out in the previous versions but shouldn't we set this to base.group_erp_manager?

Setting up the the saml provider is definitely for base.group_system but managing the user entries should be done by "Access Rights" group imo

Reply:
Yes, #652 need to ported to 17.0.